jaxrs-tck/src/main/java/jakarta/ws/rs/tck/basic/BasicResourceTest.java

```diff
@@ -0,0 +1,64 @@
+/*******************************************************************
+* Copyright (c) 2021 Eclipse Foundation
+*
+* This specification document is made available under the terms
+* of the Eclipse Foundation Specification License v1.0, which is
+* available at https://www.eclipse.org/legal/efsl.php.
+*******************************************************************/
+package jakarta.ws.rs.tck.basic;
+
+import static org.testng.Assert.assertEquals;
+
+import java.net.URI;
+import java.net.http.HttpClient;
+import java.net.http.HttpClient.Version;
+import java.net.http.HttpRequest;
+import java.net.http.HttpRequest.BodyPublishers;
+import java.net.http.HttpResponse;
+import java.net.http.HttpResponse.BodyHandlers;
+import java.util.logging.Logger;
+
+import org.jboss.arquillian.container.test.api.Deployment;
+import org.jboss.arquillian.testng.Arquillian;
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.spec.WebArchive;
+import org.testng.annotations.Test;
+
+public class BasicResourceTest extends Arquillian {
+    private final static Logger LOG = Logger.getLogger(BasicResourceTest.class.getName());
+
+    private final static String CONTEXT_ROOT = "basic";
+
+    @Deployment
+    public static WebArchive createDeployment() {
+        return ShrinkWrap.create(WebArchive.class, "basic.war")
+                         .addPackage(BasicApp.class.getPackage());
+    }
+
+    @Test
+    public void test_get() throws Exception {
```

Comment:
This does not look like java: https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html.

Reply:
I have no problems changing the method names to `testGet` or even just `get` (since we know we're in a test method).